scripts/end_to_end_demo.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+sudo snap install microk8s --classic --channel=1.18/stable
+
+sudo usermod -a -G microk8s dazwilkin
+
+sudo microk8s status --wait-ready
+
+sudo microk8s enable dns helm3 rbac
+
+echo "--allow-privileged=true" >> /var/snap/microk8s/current/args/kube-apiserver
+
+sudo microk8s stop && microk8s start
+
+sudo microk8s.kubectl label node ${HOSTNAME,,} node-role.kubernetes.io/master= --overwrite=true
+
+VERSION="v1.17.0"
+curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-${VERSION}-linux-amd64.tar.gz --output crictl-${VERSION}-linux-amd64.tar.gz
+sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
+rm -f crictl-$VERSION-linux-amd64.tar.gz
+AKRI_HELM_CRICTL_CONFIGURATION="--set agent.host.crictl=/usr/local/bin/crictl --set agent.host.dockerShimSock=/var/snap/microk8s/common/run/containerd.sock"
+
+sudo apt update
+sudo apt -y install linux-modules-extra-$(uname -r)
+sudo apt -y install dkms
+
+curl http://deb.debian.org/debian/pool/main/v/v4l2loopback/v4l2loopback-dkms_0.12.5-1_all.deb -o v4l2loopback-dkms_0.12.5-1_all.deb
+sudo dpkg -i v4l2loopback-dkms_0.12.5-1_all.deb
+sudo modprobe v4l2loopback exclusive_caps=1 video_nr=1,2
+
+sudo apt-get install -y \
+  libgstreamer1.0-0 gstreamer1.0-dev gstreamer1.0-tools \
+  gstreamer1.0-doc gstreamer1.0-plugins-base gstreamer1.0-plugins-good \
+  gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly \
+  gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools \
+  gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 \
+  gstreamer1.0-qt5 gstreamer1.0-pulseaudio 
+
+sudo gst-launch-1.0 -v videotestsrc pattern=ball ! "video/x-raw,width=640,height=480,framerate=10/1" ! avenc_mjpeg ! v4l2sink device=/dev/video1 &
+
+sudo gst-launch-1.0 -v videotestsrc pattern=smpte horizontal-speed=1 ! "video/x-raw,width=640,height=480,framerate=10/1" ! avenc_mjpeg ! v4l2sink device=/dev/video2 &
+
+sudo microk8s.helm3 repo add akri-helm-charts https://deislabs.github.io/akri/
+sudo microk8s.helm3 install akri akri-helm-charts/akri \
+  $AKRI_HELM_CRICTL_CONFIGURATION \
+  --set useLatestContainers=true \
+  --set udevVideo.enabled=true \
+  --set udevVideo.udevRules[0]='KERNEL==\"video[0-9]*\"'
+
```

Comment:
we might create a wait between the helm install and the apply streaming app to let the brokers get up and running.

something like:

```
until kubectl get pods -l akri.sh/configuration=akri-udev-video -o jsonpath='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status}{"\n"}' | grep -v ContainersReady | grep "Ready=True" | wc -l | grep 2; do echo "waiting for k3s to become ready"; sleep 10; done
```

Reply:
with maybe a better waiting message: `waiting for udev video brokers to become ready`

Reply:
i think if we wait for the brokers to come up, then the instance and configuration services will be up and ready for the streaming app to start querying its pod env vars.

Reply:
I like the idea of a wait. When I tested it out, the streaming app errored and then started running when the brokers came up, so worst case it errors briefly. But, avoiding any erroring is definitely ideal.

Reply:
Fixed: https://github.com/deislabs/akri/pull/56/commits/7c04f32209c84f59b45fe35fec49a0adab6563d5

I hope you don't mind my using `kubectl wait ...` as this seems cleaner